lambda/lambda_function.py

```diff
@@ -36,9 +37,11 @@ def can_handle(self, handler_input: HandlerInput) -> bool:
     def handle(self, handler_input: HandlerInput) -> Response:
         speak_output = "Sure, what is the question?"
 
+        # Here we are launching the skill, so we can reset the chat.
+        # Note: While testing just with RevGPT,
+        # we can see the conversation ID and parent ID always changing
+        # even when reset_chat is not called.
         chatgpt.reset_chat()
-        # Uses the session_token to get a new bearer token
-        chatgpt.refresh_session()
```

Comment:
This is not needed. Looking at the revChatGPT lib Chatbot class, we an see:

```
class Chatbot:
    """
    Initializes the chatbot

    See wiki for the configuration json:
    https://github.com/acheong08/ChatGPT/wiki/Setup

    :param config: The configuration json
    :type config: :obj:`json`

    :param conversation_id: The conversation ID
    :type conversation_id: :obj:`str`, optional
    
    :param parent_id: The parent ID
    :type parent_id: :obj:`str`, optional

    :param debug: Whether to enable debug mode
    :type debug: :obj:`bool`, optional

    :param refresh: Whether to refresh the session
    :type refresh: :obj:`bool`, optional

    :return: None or Exception
    """
    (...)
     def __init__(self, config, conversation_id=None, parent_id=None, debug=False, refresh=True) -> Exception:
     (...)
```

This is automatically part of the instantiation of the class. So I guess not needed here?